sigma/backends/elasticsearch/elasticsearch_eql.py

```diff
@@ -0,0 +1,445 @@
+            "INFORMATIONAL": 1,
+            "LOW": 21,
+            "MEDIUM": 47,
+            "HIGH": 73,
+            "CRITICAL": 99,
+        }
+
+    @staticmethod
+    def _is_field_null_condition(cond: ConditionItem) -> bool:
+        return isinstance(cond, ConditionFieldEqualsValueExpression) and isinstance(
+            cond.value, SigmaNull
+        )
+
+    def is_ip(self, value: ConditionFieldEqualsValueExpression) -> bool:
+        try:
+            ipaddress.ip_address(value)
+            return True
+        except ValueError:
+            return False
+
+    def convert_condition_field_eq_val_str(
+        self, cond: ConditionFieldEqualsValueExpression, state: ConversionState
+    ) -> Union[str, DeferredQueryExpression]:  # pragma: no cover
+        """Conversion of field = string value expressions"""
+        try:
+            if (  # Check conditions for usage of 'startswith' operator
+                self.startswith_expression
+                is not None  # 'startswith' operator is defined in backend
+                and cond.value.endswith(
+                    SpecialChars.WILDCARD_MULTI
+                )  # String ends with wildcard
+                and not cond.value[
+                    :-1
+                ].contains_special()  # Remainder of string doesn't contains special characters
+            ):
+                expr = (
+                    self.startswith_expression
+                )  # If all conditions are fulfilled, use 'startswith' operartor instead of equal token
+                value = cond.value[:-1]
+            elif (  # Same as above but for 'endswith' operator: string starts with wildcard and doesn't contains further special characters
+                self.endswith_expression is not None
+                and cond.value.startswith(SpecialChars.WILDCARD_MULTI)
+                and not cond.value[1:].contains_special()
+            ):
+                expr = self.endswith_expression
+                value = cond.value[1:]
+            elif (  # contains: string starts and ends with wildcard
+                self.contains_expression is not None
+                and cond.value.startswith(SpecialChars.WILDCARD_MULTI)
+                and cond.value.endswith(SpecialChars.WILDCARD_MULTI)
+                and not cond.value[1:-1].contains_special()
+            ):
+                expr = self.contains_expression
+                value = cond.value[1:-1]
+            elif (  # wildcard match expression: string contains wildcard
+                self.wildcard_match_expression is not None
+                and cond.value.contains_special()
+            ):
+                expr = self.wildcard_match_expression
+                value = cond.value
+            elif (  # Use '==' as operator for empty string or ip addresses
+                cond.value.convert() == "" or self.is_ip(cond.value)
+            ):
+                expr = "{field}" + "==" + "{value}"
+                value = cond.value
```

Comment:
Except this if-statement the code is copy&paste from the backend base class. The override should better only contain this if block that also incorporates the return statement at the botton and an else block that returns the result of the base class implementation called with `super()`.